platform/disk/ephemeral_device_partitioner_test.go

```diff
@@ -0,0 +1,424 @@
+package disk_test
+
+import (
+	"errors"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	fakeboshaction "github.com/cloudfoundry/bosh-agent/agent/action/fakes"
+	. "github.com/cloudfoundry/bosh-agent/platform/disk"
+	"github.com/cloudfoundry/bosh-agent/platform/disk/fakes"
+	boshlog "github.com/cloudfoundry/bosh-utils/logger"
+	fakesys "github.com/cloudfoundry/bosh-utils/system/fakes"
+)
+
+var _ = Describe("EphemeralDevicePartitioner", func() {
+	var (
+		devicePath string
+
+		diskUtil   *fakes.FakeDiskUtil
```

Comment:
Our go compiler yells because this isn't referenced any more. Can you remove it?